Embedded Metadata.js

```diff
@@ -892,9 +932,10 @@ function getAuthorFromByline(doc, newItem) {
  */
 function finalDataCleanup(doc, newItem) {
 	if (typeof newItem.tags == 'string') {
+		Z.debug("newItem.tags is a string");
 		newItem.tags = [newItem.tags];
 	}
-	if (newItem.tags && newItem.tags.length && Zotero.parentTranslator) {
```

Comment:
not sure what `Zotero.parentTranslator` was checking for here—I'd think this should run whenever EM is finding a tag string to parse, not only when it's the parent? Unless the point is to only run when it has a child translator so the child translator can choose what to do with the sometimes garbage tags?